kube/services/sheepdog/sheepdog-deploy.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: sheepdog-deployment
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+  template:
+    metadata:
+      labels:
+        app: sheepdog
+    spec:
+      volumes:
+        - name: config-volume
+          secret:
+            secretName: "sheepdog-secret"
+        - name: cert-volume
+          secret:
+            secretName: "cert-sheepdog-service"
+        - name: ca-volume
+          secret:
+            secretName: "service-ca"
+      containers:
+        - name: sheepdog
+          image: quay.io/cdis/sheepdog:master
+          livenessProbe:
+            httpGet:
+              path: /_status
+              port: 80
+            initialDelaySeconds: 30
+            periodSeconds: 60
+            timeoutSeconds: 30
+          ports:
+          - containerPort: 80
+          - containerPort: 443
+          env:
+          - name: DICTIONARY_URL
+            valueFrom:
+              configMapKeyRef:
+                name: global
+                key: dictionary_url
```

Comment:
Can we set this to the datadictionary url so the service will work when it first comes up?

Reply:
Yeah - exactly - we set the dictionary_url in the configmap - terraform has a dictionary_url variable too ...